symforce/codegen/format_util.py

```diff
@@ -113,3 +112,46 @@ def format_py_dir(dirname: T.Openable) -> None:
         env=dict(os.environ, RUFF_NO_CACHE="true"),
         text=True,
     )
+
+_rustfmt_path: T.Optional[Path] = None
+
+def _find_rustfmt() -> Path:
```

Comment:
Is it acceptable to add a dependency on rustfmt like this? It makes me uncomfortable to have a system dependency but unsure how to add it otherwise.